src/qml/MapCanvas.qml

```diff
@@ -33,6 +33,7 @@ Item {
   property bool hovered: false
   property bool pinched: pinchHandler.active
   property bool freehandDigitizing: false
+  property bool isRotationFeaturesRequested: false
```

Comment:
I am extremely sensitive to touching MapCanvas and its handlers, can we revert all changes here please.

If you are fixing something that's not related to feature rotation, it's most welcome, but please do so in a separate PR. :)